x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/session/EsqlSessionTests.java

```diff
@@ -45,7 +45,7 @@ public void testCreateIndexExpressionFromAvailableClusters() {
             executionInfo.swapCluster(remote1Alias, (k, v) -> new EsqlExecutionInfo.Cluster(remote1Alias, "*", true));
             executionInfo.swapCluster(remote2Alias, (k, v) -> new EsqlExecutionInfo.Cluster(remote2Alias, "mylogs1,mylogs2,logs*", true));
 
-            String indexExpr = EsqlSession.createIndexExpressionFromAvailableClusters(executionInfo);
+            String indexExpr = CcsUtils.createIndexExpressionFromAvailableClusters(executionInfo);
```

Comment:
In a follow on PR, I'll remove this test and create a CCSUtils test to house these tests.

Reply:
Thanks @quux00  - hope I didn't waste too much of your time with the duplicate PR. I ended up doing them while working on the merging conflicts and made the pings hoping to draw your attention before too long.

I'll try to get this is in ASAP so you can continue to add your own modification in.